svix-webhooks/values.cue

```diff
@@ -0,0 +1,26 @@
+// Code generated by timoni.
+// Note that this file must have no imports and all values must be concrete.
+
+@if(!debug)
+
+package main
+
+// Defaults
+values: {
+	image: {
+		repository: "docker.flipt.io/svix/svix-server"
+		digest:     "sha256:1a316679091323636fadacc0930a7a0ef0f05ceeea7bda0e8aab7cb4e4704f19"
+		tag:        "v1.18"
+	}
+	test: image: {
+		repository: "busybox"
+		digest:     ""
+		tag:        "latest"
+	}
+
+	// These values should be known before hand.
+	connections: {
+		redisDSN: "redis://redis:6379"
+		dbDSN:    "postgres://postgres:5432"
+	}
```

Comment:
I reckon you could just drop these defaults and let it fail fast if they don't provide them.

Reply:
Hmmm maybe timoni mod vet screams if these are set. So maybe ignore me.

Reply:
@GeorgeMac Yeah that is why I made those defaults, but there might be a better way to do this type of pattern. I want those required but the defaults aren't great 😞